content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,248 @@
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">
+      <p className="tw-m-0 tw-mr-2">Docs</p>
+      <OpenInNewIcon className="tw-inline" size={18} />
+  </a>
+
+
+### 💖 Advanced admin capabilities
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+Sourcegraph 4.0 supports multi-version upgrades so you can upgrade to 4.0 directly from version 3.20 or higher. It eliminates the number of steps and the amount of time it takes to get access to new features and enhancements released each month. 
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20upgrades%20final',
+    mp4: 'blog/release-post/4.0/Multi-version%20upgrades%20final'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Demo of multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center">
+      <p className="tw-m-0 tw-mr-2">Docs</p>
+      <OpenInNewIcon className="tw-inline" size={18} />
+  </a>
+
+
+<a href="https://docs.sourcegraph.com/admin/analytics"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### View usage and measure the value of our platform with new and enhanced in-product analytics
+Our new in-product analytics help admins to understand user engagement, measure efficiency in terms of time saved, and calculate the value of Sourcegraph and its features. 
+
+These analytics work across all of our products and provides ROI-based metrics such as:
+
+- The number of searches developers have run in Sourcegraph, and how many hours of development time have been saved as a result.
+- How many code insights have been created, viewed, and interacted with.
+- The number of changesets that have been created and merged via batch changes, and how much time has been saved as a result.
+- How frequently code navigation has been used by developers to more quickly understand code.
+- How often IDE and Browser extensions are used across your organization, and how much time has been saved as a result.
+- Measure precise code navigation coverage and access precise graph data configured for all of your repositories.
+
+Admins can configure the date range selector and view the adoption of a feature over time by both volume and the number of users. The value calculators use data derived from the event logs and are designed to allow admins to adjust to conditions in their environment by editing the default time-saved values. 
```

Comment:
This is a bit confusing - basically we are trying to say you can customize the assumptions to align with how you're using the product.